src/main/java/seedu/trackascholar/model/util/SampleDataUtil.java

```diff
@@ -22,22 +22,22 @@ public static Applicant[] getSamplePersons() {
         return new Applicant[] {
             new Applicant(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Scholarship("NUS Global Merit Scholarship"), new ApplicationStatus("pending"),
-                getTagSet("friends")),
+                getMajorSet("friends")),
```

Comment:
I think the example here also have to change?

Reply:
Ok thank you it has been resolved.